dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -111,9 +117,10 @@ const brandExtensionLink = css`
 `;
 
 export const Columns: React.FC<{
+	display: ArticleDisplay;
```

Comment:
For consistency, can we change this to `format`? I think it's perfectly fine to pass `display` by itself in general, but we just have a convention of always passing `format`. I guess the win is it's more scalable

Reply:
Ah, yes - thanks, I'll update :)